### DISCUSSION TITLE
decimal part of widget very long

### DISCUSSION THREAD

**Discussion by zippermonkey** [Ivy-Apps/ivy-wallet#3265] [Open problems]
![Screenshot_2024-07-05-10-28-04-463_com.miui.home-edit.jpg](https://github.com/Ivy-Apps/ivy-wallet/assets/41142057/c2055841-ab14-4b85-8823-df2120adab4c)

**Comment by ILIYANGERMANOV** [comment]
Hey @zippermonkey what Ivy Wallet version do you use? Can you try the latest one from GitHub?
